examples/data-objects/shared-text/src/index.ts

```diff
@@ -69,7 +66,7 @@ class SharedTextFactoryComponent extends RuntimeFactoryHelper implements IFluidD
             ],
             buildRuntimeRequestHandler(
                 defaultRouteRequestHandler(DefaultComponentName),
-                innerRequestHandler,
+                rootDataStoreRequestHandler,
```

Comment:
As opposed to here seems needed in order to request the _scheduler in component.ts, so keep this one